### ISSUE TITLE
Settings  - Input. Clicking wrong button to save values

### ISSUE THREAD

**Issue by paaljoachim** [joedolson/my-calendar#492]
**Describe the bug**
I just made a mistake. I adjusted the Location Controls and I kept clicking the button above "Save Input Settings" to save the values. Without really thinking about which blue button I was clicking to save.

**To Reproduce**
1. Go to Settings -> Input.
2. Add some values to the Location Control fields.
3. Click "Save Input Settings" to save the values.
4. Notice the values added to the Location Controls are gone after having clicked the Save Input Settings button.

**Expected behavior**
That the values would not disappear. 

**Screenshots**
It is very easy to go up and click the blue button without thinking about the Update Location Controls button below the Location Controls fields.

![Screenshot 2022-12-11 at 22 41 24](https://user-images.githubusercontent.com/5323259/206955764-9ee3c122-433e-4a8c-9ec1-fd9ff5876d1b.jpg)

I am not sure we could/should do anything here. It might take a few tries before the user hopefully notices the Update Location Controls button on the bottom of the page.

**Comment by paaljoachim** [comment]
I downloaded the latest Github build.
On my test site running Twenty Twenty-Three and WordPress 6.1.1
I changed some of the Location Controls and clicked the Update Location Controls button, but the values went back to the values I had earlier before I changed these.

Example I changed the Controls for Country (actually any of the fields had the same effect) from Norway,Norway to usa,USA clicked Update Location Controls and it went back to Norway,Norway.
